OPS/load_stage.sql

```diff
@@ -0,0 +1,275 @@
+/**************************************************************************
+* Copyright 2020 Observational Health Data Sciences and Informatics (OHDSI)
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+* 
+* Authors: Eduard Korchmae, Timur Vakhitov, Dmitry Dymshyts, Christian Reich
+* Date: 2020
+**************************************************************************/
+
+--0. Latest update construction
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'OPS',
+	pVocabularyDate			=> TO_DATE ('20200101', 'yyyymmdd'),
```

Comment:
Is there any way to automate the date and version?

Reply:
Once we integrate automated file retrieval